common/src/main/java/fi/hel/verkkokauppa/common/rest/CommonServiceConfigurationClient.java

```diff
@@ -134,6 +136,36 @@ public String getMerchantConfigurationValue(String merchantId, String namespace,
         }
     }
 
+    public MerchantDto updateMerchantConfigurationValueByKey(String merchantId, String namespace, String key, String value) {
```

Comment:
Why this is added to CommonServiceConfigurationClient?

Reply:
If this is needed only in tests, then this should be moved to another class.

Reply:
I added it here because it is a valid method which enables the modification of merchant configurations for all core services. But at the moment it is only needed in the funcitonal test cases. I can just move it to the test class where it is needed, I dont think it needs a class of its own. If this kind of functionality is needed in the future for other core services, then we can move it back here to CommonServiceConfigurationClient.